File: CSharp/OpenTraceability/Models/Events/EventBusinessTransaction.cs
using OpenTraceability.Utility.Attributes;

namespace OpenTraceability.Models.Events
{
    public enum EventBusinessTransactionType
    {
        // TODO: add in the default business transaction types
    }

    public class EventBusinessTransaction
    {
        [OpenTraceabilityJson("type")]
        [OpenTraceability("@type")]
        public Uri Type { get; set; }

        [OpenTraceabilityJson("bizTransaction")]
        [OpenTraceability("text()")]
        public string Value { get; set; }
    }
}

File: JDK11/opentraceability/src/main/java/opentraceability/models/events/eventbusinesstransaction.java
package opentraceability.models.events;

import opentraceability.utility.attributes.OpenTraceabilityAttribute;
import opentraceability.utility.attributes.OpenTraceabilityJsonAttribute;

import java.net.URI;

public class EventBusinessTransaction {

    @OpenTraceabilityJsonAttribute(name="type")
    @OpenTraceabilityAttribute("", "@type")
    public URI type;

    @OpenTraceabilityJsonAttribute(name="bizTransaction")
    @OpenTraceabilityAttribute("", "text()")
    public String value;

}